pkg/operator/ceph/nfs/nfs.go

```diff
@@ -293,14 +293,17 @@ func validateGanesha(context *clusterd.Context, clusterInfo *cephclient.ClusterI
 // create and enable default RADOS pool
 func (r *ReconcileCephNFS) createDefaultNFSRADOSPool(n *cephv1.CephNFS) error {
 	poolName := n.Spec.RADOS.Pool
-	// Settings are not always declared and CreateReplicatedPoolForApp does not accept a pointer for
```

Comment:
If rook doesn't create this pool, will ceph create it automatically? If so, seems like we could stop creating it explicitly since after #9209 is available, they should be creating the .nfs pool with the pool CR.

Reply:
I don't believe ceph creates it automatically, no. At least it doesn't for all versions.

The operation is idempotent though AFAIK. If the pool exists, Ceph won't fail the command. Same for setting the application. Since the command doesn't specify a crush rule, I do wonder if it might reset the pool to defaults or not. That would be an integration we'd have to investigate a bit I think.

Reply:
Ok, let's leave this for now to ensure the pool is created by default. With #9209 perhaps we require that they define the .nfs CephBlockPool.